API.md

```diff
@@ -204,7 +204,7 @@ Declares a subscription path client can subscribe to where:
         - `params` - the parameters parsed from the subscription request path if the subscription
           path definition contains parameters.
         - `next` - the continuation method required to complete the unsubscribe request using the
-          signature `function()`
+          signature `function().`
```

Comment:
Interesting -- I'm seeing the period outside of the quotes when I view the changes:
<img width="853" alt="screen shot 2016-09-25 at 2 43 06 pm" src="https://cloud.githubusercontent.com/assets/6673263/18817448/73122e80-832e-11e6-8123-6cb90a5f755b.png">

<img width="794" alt="screen shot 2016-09-25 at 2 45 49 pm" src="https://cloud.githubusercontent.com/assets/6673263/18817463/d032a9be-832e-11e6-8405-efd75647ba69.png">